owncloudApp/src/main/res/values/strings.xml

```diff
@@ -692,4 +692,10 @@
     <string name="release_notes_title_6">Hidden files</string>
     <string name="release_notes_subtitle_6">Added a new option in settings to decide to show or not hidden files.</string>
 
+    <string name="release_notes_title_7">Sharing from inside</string>
+    <string name="release_notes_subtitle_7">Added the sharing option to folders from the list of its contents</string>
+
+    <string name="release_notes_title_8">Bugfixing</string>
+    <string name="release_notes_subtitle_8">Some existing bugs were fixed</string>
```

Comment:
Not 100% sure about this one, but ok

Reply:
Well, not an specific one like the others. But, fixes are also part of the release.